src/main/java/com/egen/ordermanagement/utility/EmailSender.java

```diff
@@ -0,0 +1,50 @@
+package com.egen.ordermanagement.utility;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.util.Properties;
+
+public class EmailSender {
```

Comment:
It was given to show you the example of how to process consumed records.

Reply:
But I was trying to see if it works with a fake SMTP using mailtrap.io I accidentally pushed this filed as well